xbmc/input/linux/CMakeLists.txt

```diff
@@ -1,7 +1,14 @@
-set(SOURCES LinuxInputDevices.cpp
-            LIRC.cpp)
+set(SOURCES "")
+set(HEADERS "")
 
-set(HEADERS LinuxInputDevices.h
-            LIRC.h)
+if(LIRC_DEVICE)
+  list(APPEND SOURCES LIRC.cpp)
+  list(APPEND HEADERS LIRC.h)
+endif()
+
+if(CORE_PLATFORM_NAME_LC STREQUAL rbpi OR CORE_PLATFORM_NAME_LC STREQUAL gbm OR CORE_PLATFORM_NAME_LC STREQUAL aml)
```

Comment:
CORE_SYSTEM_NAME STREQUAL linux

Reply:
@wsnipex this should only be built for these platforms. If you look at `CWinEventsLinux` you will see why :smile_cat: